src/cgnslib.c

```diff
@@ -48,6 +48,7 @@ freely, subject to the following restrictions:
 # define UNLINK unlink
 #endif
 
+#include "cgnsconfig.h"
```

Comment:
What is the reason for adding cgnsconfig.h? It's not needed.

Reply:
`cgnstypes.h` requires `CG_BUILD_HDF5`'s value to be available. `cgnsconfig.h` defines it, but it was not always included before `cgnstypes.h`.

Reply:
This is probably contingent on the other discussion.